app/routes/joblistings/components/JoblistingEditor.tsx

```diff
@@ -280,6 +281,12 @@ const JoblistingEditor = () => {
                 component={DatePicker.Field}
               />
             </RowSection>
+            <Field
+              name="rollingRecruitment"
+              label="Løpende Opptak"
+              type="checkbox"
+              component={CheckBox.Field}
```

Comment:
Perhaps this should be a Switch? 
```suggestion
              component={Switch.Field}
```

Reply:
Done.